libs/designsystem/src/lib/helpers/line-clamp-helper.ts

```diff
@@ -0,0 +1,10 @@
+import { Injectable } from '@angular/core';
+
+@Injectable()
+export class LineClampHelper {
+  setLineClamp(el: HTMLElement, maxLines: number, lineHeight: string): void {
+    el.style.setProperty('--line-clamp', `${maxLines}`);
+    el.style.setProperty('--line-height', lineHeight);
+    el.classList.add('kirby-line-clamp');
+  }
+}
```

Comment:
Maybe a bit bloated having a dedicated provider for just setting 3 lines of css? 🤔

Reply:
Yeah, perhaps. We discussed it when the PR was in draft. I want to keep line clamp separate from the auto-resize-text functionality in the fit-headings directive. Do you have a suggestion for an alternative?